advanced/pdf_grayscale_transform.go

```diff
@@ -51,34 +51,14 @@ func main() {
 }
 
 func convertPdfToGrayscale(inputPath, outputPath string) error {
-	f, err := os.Open(inputPath)
-	if err != nil {
-		return err
-	}
-
-	defer f.Close()
+	readerOpts := model.NewReaderOpts()
+	readerOpts.LazyLoad = false
```

Comment:
Should just use the default options (nil) everywhere except where otherwise needed.  Everyone should generally be using lazy loadniig by default.